kube-derive/src/custom_resource.rs

```diff
@@ -270,6 +281,7 @@ impl CustomDerive for CustomResource {
             #visibility struct #rootident {
                 #visibility api_version: String,
                 #visibility kind: String,
+                #schemars_skip
```

Comment:
We should probably document how we mangle the schemars attributes (and why)

Reply:
I originally excluded it because it wasn't possible to generate a schema with it (`ObjectMeta` not implenting `JsonSchema`). But I noticed

> CustomResources store structured data in custom fields (alongside the built-in fields `apiVersion`, `kind` and `metadata`, which the API server validates implicitly).
>
> [Extend the Kubernetes API with CustomResourceDefinitions | Kubernetes](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#specifying-a-structural-schema)

So I excluded these and added a comment about it.